src/cli/flags/repo.go

```diff
@@ -12,18 +12,20 @@ const (
 	AWSSessionTokenFN    = "aws-session-token"
 
 	// Corso Flags
-	CorsoPassphraseFN = "passphrase"
-	SucceedIfExistsFN = "succeed-if-exists"
+	CorsoPassphraseFN       = "passphrase"
+	UpdateCorsoPassphraseFN = "new-passphrase"
```

Comment:
Var names should mimic the flag value whenever possible.
```suggestion
	PassphraseFN       = "passphrase"
	NewPassphraseFN = "new-passphrase"
```